src/components/MapsSection/MapsSection.tsx

```diff
@@ -37,7 +37,7 @@ const MapsSection = ({
   const [isLoading, setLoading] = useState<boolean>(false);
   const [currentVisu, setCurrentVisu] = useState<IEEInfo>(defaultEEInfo);
   const [imageData, setImageData] = useState<IMapInfo>({
-    name: "cisterna",
+    name: "default",
```

Comment:
Instead of using "Default" here I think it would be better if we added an empty string so no one gets confused on thinking there's a raster with the id "Default"

```suggestion
    name: "",
```

Reply:
sure! i'll take care of it! 
thx!

Reply:
solved!